packages/mobile/src/screens/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -1,26 +1,46 @@
 import React, { FC } from 'react';
-import { Button, Text, View } from '~/components/components';
-import { useTranslation } from '~/hooks/hooks';
+import { Button, View, Input } from '~/components/components';
+import { UserSignInDto } from '@vse-bude/shared';
+import { useAppForm, useTranslation } from '~/hooks/hooks';
+import { signIn } from '~/validation-schemas/validation-schemas';
+import { DEFAULT_SIGN_IN_PAYLOAD } from './common/constants';
 
 type Props = {
-  onSubmit: () => void;
+  onSubmit: (payload: UserSignInDto) => void;
 };
 
-const SignInForm: FC<Props> = () => {
+const SignInForm: FC<Props> = ({ onSubmit }) => {
+  const { control, errors, handleSubmit } = useAppForm<UserSignInDto>({
+    defaultValues: DEFAULT_SIGN_IN_PAYLOAD,
+    validationSchema: signIn,
+  });
   const { t } = useTranslation();
 
   return (
-    <>
-      <Text>{t('verification.SING_IN')}</Text>
+    <View>
```

Comment:
Additional wrapper seems to be redundant

Reply:
corrected